server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -1433,8 +1432,10 @@ public void loadTablet(TInfo tinfo, TCredentials credentials, String lock,
             all.remove(extent);
 
             if (!all.isEmpty()) {
-              log.error("Tablet {} overlaps previously assigned {} {} {}", extent,
-                  unopenedOverlapping, openingOverlapping, onlineOverlapping + " " + all);
+              log.error(
+                  "Tablet {} overlaps a previously assigned tablet. It is possibly due to a recent split. "
+                      + "Possible overlapping tablets:  Unopened: {}, Opening: {}, Online: {}",
+                  extent, unopenedOverlapping, openingOverlapping, onlineOverlapping);
```

Comment:
I think these are definitely overlapping, not just possibly overlapping
```suggestion
                      + "Overlapping tablets: Unopened: {}, Opening: {}, Online: {}",
                  extent, unopenedOverlapping, openingOverlapping, onlineOverlapping);
```